rhea/system/clock.py

```diff
@@ -46,6 +46,24 @@ def frequency(self, f):
     def period(self):
         return self._period
 
+    @property
+    def ticks(self):
+        """
+        Returns the number of simulation ticks that will elapse per clock tick.
+        """
+        return self._ticks
+
+    def delay(self, count=1):
```

Comment:
We discussed this briefly in the issue, I don't like the name `delay` for this generator.  In myhdl signals currently have a `posedge` and `negedge` events, what might read well is to use the plural of these:

```python
    yield clock.posedges(10)
    yield clock.negedges(5)
```

or the "edge" could be an optional argument

```python
    yield clock.edges(10)            # default is posedge, edge=True
    yield clock.edges(5, False)   # edge=False
```